content/docs/authorization/v2.0 Tech Preview/configuration/powerflex/_index.md

```diff
@@ -0,0 +1,117 @@
+    ```
+
+3. Create the proxy-server-root-certificate secret.
+
+    If running in *insecure* mode, create the secret with empty data:
+
+      ```bash
+      
+      kubectl -n vxflexos create secret generic proxy-server-root-certificate --from-literal=rootCertificate.pem= -o yaml --dry-run=client | kubectl apply -f -
+      ```
+
+    Otherwise, create the proxy-server-root-certificate secret with the appropriate file:
+
+      ```bash
+      
+      kubectl -n vxflexos create secret generic proxy-server-root-certificate --from-file=rootCertificate.pem=/path/to/rootCA -o yaml --dry-run=client | kubectl apply -f -
+      ```
+
+4. Prepare the driver configuration secret, applicable to your driver installation method, to communicate with the CSM Authorization sidecar.
+
+    **Operator**
+
+    Refer to the [Create Secret](../../../../deployment/csmoperator/drivers/powerflex/#create-secret) section to prepare `secret.yaml` to configure the driver to communicate with the CSM Authorization sidecar.
+
+    - Update `endpoint` to match the localhost endpoint in `samples/secret/karavi-authorization-config.json`.
+
+    - Update `skipCertificateValidation` to `true`.
+
+    - The `username` and `password` can be any value since they will be ignored.
+
+    Example:
+
+    ```yaml
+    - username: "ignored"
+      password: "ignored"
+      systemID: "ID2"
+      endpoint: "https://localhost:9400"
+      skipCertificateValidation: true 
+      isDefault: true 
+      mdm: "10.0.0.3,10.0.0.4"
+    ```
+
+5. Enable CSM Authorization in the driver installation applicable to your installation method.
+
+    **Operator**
+
+    Refer to the [Install Driver](../../../../deployment/csmoperator/drivers/powerflex/#install-driver) section to edit the parameters in the Custom Resource to enable CSM Authorization.
+
+    Under `modules`, enable the module named `authorization`:
+
+    - Update the `enabled` field to `true.`
+
+    - Update the `image` to the image of the CSM Authorization sidecar. In most cases, you can leave the default value.
+
+    - Update the `PROXY_HOST` environment value to the hostname of the CSM Authorization Proxy Server. `csm-authorization.com` is a placeholder for the proxyHost. See the administrator of CSM for Authorization for the correct value.
+
+    - Update the `SKIP_CERTIFICATE_VALIDATION` environment value to `true` or `false` depending on if you want to disable or enable certificate validation of the CSM Authorization Proxy Server.
+
+    Example: 
+
+    ```yaml
+    modules:
+      # Authorization: enable csm-authorization for RBAC
+      - name: authorization
+        # enable: Enable/Disable csm-authorization
+        enabled: true
+        configVersion: v1.10.0
+        components:
+        - name: karavi-authorization-proxy
+          image: dellemc/csm-authorization-sidecar:v1.10.0
+          envs:
+            # proxyHost: hostname of the csm-authorization server
```

Comment:
The data and comments here should match the sample file:

https://github.com/dell/csm-operator/blob/authorization-gitops/samples/authorization/csm_authorization_proxy_server_v200-alpha.yaml

There have been changes where these fields are now in the components section, not envs.

Reply:
The additional parameters that you are referring to are for CSM Authorization installation which I added: https://github.com/dell/csm-docs/pull/1101/files#diff-0748aa8581f546d84d13a257ac31129abb217e374b302da45a394762e48b5cbcR88.

I didn't see any additional parameters for the sidecar.

Reply:
You're right. My misake.